rules-tests/Instanceof_/Rector/Ternary/FlipNegatedTernaryInstanceofRector/Fixture/correct_case.inc

```diff
@@ -0,0 +1,32 @@
+<?php
```

Comment:
should be `.php.inc` instead of `.inc` extension

Reply:
Commited by accident, thanks :+1: